docs/source/concepts/CrystalField.rst

```diff
@@ -0,0 +1,603 @@
+and so can expanded in terms of spherical harmonic functions
+
+.. math::
+    V_{\mathrm{CF}}(\mathbf{r}_i) = \sum_{lm} A_l^m r^l Y_{lm}(\mathbf{r}_i),
+  :label: cfexp
+
+where the radial part is
+
+.. math::
+    A_l^m = \frac{1}{4\pi\epsilon_0} \left\{ (-1)^m \frac{4\pi}{2l+1} \sum_j \frac{q_j}{r_j^{l+1}} Y_{l,-m}(\mathbf{r}_j) \right\}. 
+  :label: alm
+
+The above equations define the *point charge model* `[Hutchings64]`_.
+In many solids, though, the point charge approximation breaks down,
+because the ligand electrons may be involved in bonding or charge transfer processes,
+and thus may have a spatial extent and may no longer be treated as point charges situated at the atomic sites.
+The formulism detailed above, however, was still found to be applicable to a wide variety of materials,
+*if* the radial part :math:`A_l^m r^l` is treated as a variable *crystal field parameter*.
+This is because the spherical harmonic functions form a complete orthogonal basis set,
+so that any function may be expressed as an expansion in them.
+Values obtained from equation :eq:`alm`, however, do not generally match experimentally determined crystal field parameters,
+except for some insulators and if the point charges :math:`q_j` are scaled from their nominal valences.
+
+Mantid contain routines which can calculate the :math:`A_l^m` parameters
+which may be used as starting parameters for a fit to inelastic neutron scattering data.
+
+
+Crystal Field Energy Levels
+---------------------------
+
+The above potential, being an electric interaction, acts only on the orbital part of the electronic wavefunctions.
+Its effect is to lift the degeneracy of the orbital states 
+(labelled by the secondary orbital angular momentum quantum number :math:`L_z`)
+of the magnetic ion :math:`i` giving rise to a set of energy levels.
+
+At zero temperature only the ground state energy level is occupied by the electrons of the magnetic ion
+but as the temperature increases, higher energy excited levels will become occupied.
+The different levels have different magnetic moment expectation values,
+which leads to a change of the magnetic susceptibility of materials containing this magnetic ion with temperature
+as the population of the levels change.
+Because different magnetic materials have different crystal structures and hence a different local crystalline
+environment around the magnetic ions, they will also have different susceptibilities,
+which is determined in part by the crystal field.
+An applied magnetic field will change the energies of the levels with respects to each other (the Zeeman splitting)
+so the crystal field will also affect the magnetisation (the magnetic moment as a function of applied magnetic field).
+
+Inelastic neutron spectroscopy can measure the energy difference between these crystal field energy levels
+as neutrons may excited or de-excited electrons occupying one level to another.
```

Comment:
I think there is a typo in this sentence

Reply:
yes... thanks for spotting!